docs/source/guide/iam-example-managing-access-keys.rst

```diff
@@ -128,7 +127,6 @@ Example
 
     import boto3
 
-
```

Comment:
nit; while these save a little space, they're technically taking correctly formatted code and breaking the style guide. This will likely result in new errors being thrown by linters. Was the reason for the change primarily around readability?

Reply:
Breaking which style guide? It must be a different one from what I've been looking at. I removed them mostly because it seemed from what I'd read that they're considered "legal but not typical" and since they didn't appear to be serving any useful purpose, and I didn't see anything to the contrary, I did start changing them. But I didn't change them all, either. I'll leave them alone when I redo the PR.

Reply:
E302 would be the code it's under I believe. I think the line I put this comment on is actually correct, but the others with a `def` or `class` definition need 2 lines between top level elements in a file. It's a bit silly, but my main concern is automation doing validation, and if we're going to tweak these it should be uniform.

Reply:
OK, I misunderstood what your concern was with my first comment (probably obviously). I think these inconsistencies were already present, though I probably added a couple of new ones.

But yes, basically this is a space saving thing; vertical screen space is at a premium and finding ways that let you see more (and more context) without significant negative impact on the readability of the content is pretty important, so I thought nothing of it. If this is a problem for linters, of course, I'll try to find any of these I changed and change them back, though doing so may be... an adventure. :)